tests/testthat/test-remove_constants.R

```diff
@@ -25,5 +25,5 @@ test_that("remove_constants works", {
   expect_identical(report[["empty_columns"]], "empty_column")
   expect_identical(report[["constant_columns"]],
                    "event_name, country_code, country_name")
-  expect_identical(report[["empty_rows"]], 10L)
+  expect_identical(report[["empty_rows"]], 7L)
```

Comment:
@Karim-Mane, can you double check that this was a bug and we are indeed expecting 7 here?

Reply:
@Bisaloo - The expected value is actually `10`, and that is what I have on current version (at least on the fix-issue-151 branch).

Many of such issues have been fix along #157.